smartmin/users/models.py

```diff
@@ -4,8 +4,13 @@
 from django.conf import settings
 from django.contrib.auth.hashers import check_password
 from django.db import models
+from django.db.models.signals import post_migrate
 from django.utils import timezone
 
+from .perms import sync_permissions
+
+post_migrate.connect(sync_permissions)
```

Comment:
This will now require `smartin.users` to be in the installed apps to work.
I guess there might be someone using smartmin without that app

can that be moved to `smartmin/__init__.py`?

Reply:
I forgot that `smartmin` is an app too at the root level... will move